<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationModule</class>
 <widget class="QWidget" name="CalibrationModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>520</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IR CAM 2 (RIGHT)</string>
   </property>
  </widget>
  <widget class="QLabel" name="irRightLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>512</width>
     <height>288</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius: 25px; background: white;</string>
   </property>
   <property name="text">
    <string>IR RIGHT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="serialNum">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rightStart">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>650</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB CAM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startStreaming">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>110</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>All Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rgbStart">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>↑IR CAM 1(LEFT)</string>
   </property>
  </widget>
  <widget class="QLabel" name="depthLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>512</width>
     <height>288</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius: 25px; background: white;</string>
   </property>
   <property name="text">
    <string>DEPTH</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="rgbCaptureList">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>240</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="irLeftLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>512</width>
     <height>288</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius: 25px; background: white;</string>
   </property>
   <property name="text">
    <string>IR LEFT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="rgbCapture">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[System]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftStart">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>650</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="captrueImage">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>140</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>All Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceFirmware_2">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>시리얼 넘버    : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="startCalibration">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>140</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate!!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rgbStop">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>370</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IR CAM 1 (LEFT)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>650</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>↑IR CAM 2(RIGHT)</string>
   </property>
  </widget>
  <widget class="QLabel" name="camName">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>↑RGB CAM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopStreaming">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>110</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>All Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceName">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>연결된 장치    : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>↑DEPRH MAP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftStop">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>650</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="rgbCount">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0/20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="firmwareVer">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="leftCount">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0/20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="leftCapture">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>650</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceFirmware">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>카메라 펌웨어 : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rgbLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>512</width>
     <height>288</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius: 25px; background: white;</string>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rightCount">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>520</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0/20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>210</y>
     <width>241</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
border: 2px solid gray;
background-image: url(:/myImage.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="ir1CaptureList">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>390</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rightCapture">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>650</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ir2CaptureList">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>540</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>180</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>message!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightStop">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>650</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
